python/test/unit/language/test_matmul.py

```diff
@@ -921,8 +926,8 @@ def create_operand(dtype: str, size0: int, size1: int, k_dim: int, transpose: bo
     a, a_ref = create_operand(A_DATA_TYPE, M, K, 1)
     b, b_ref = create_operand(B_DATA_TYPE, K, N, 0, B_TRANS)
 
-    a_scale_mxfp4 = MXScaleTensor(size=(M, (K + 32 - 1) // 32), device=device).random(high=64.0)
-    b_scale_mxfp4 = MXScaleTensor(size=(N, (K + 32 - 1) // 32), device=device).random(high=64.0)
+    a_scale_mxfp4 = MXScaleTensor(size=(M, (K + 32 - 1) // 32), device=device).random(high=32.0)
```

Comment:
Why changing the `high` here?

Reply:
It failed on one test which highly likely due to overflow. The characteristics includes: only 1 element mismatches, differece is 0.0x scale, and the test passes if change random seed. Here 64 means the scale is 64, which likely causes overflow. So I adjusted it a little bit.